src/styles/blocks/review.scss

```diff
@@ -0,0 +1,9 @@
+.review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+
+  &__count {
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used but not defined in this file. Ensure that it is defined or imported from another SCSS file to avoid compilation errors.